scripts/CsvToJson.js

```diff
@@ -23,6 +23,11 @@ class CsvToJsonScript extends Base {
     const isValid = this.validate(program, this.message, this.params);
     if (isValid.flag) return console.log(isValid.output);
 
+    // Get any extra arguments and pass as params to csvToJson child process
+    const args = program.rawArgs;
+    const rest = args.slice(7);
+    const moduleParams = rest.length > 0 ? rest.join(' ') : '';
```

Comment:
More of a  style preference, but can just `!rest.length`

Reply:
I like rest.length > 0 better because it's explicit. I always stay away from boolean casts. What !rest.length indicates to the reader: If length property is falsy (= one of false/undefined/empty string/0). while rest.length > 0 express a clearer intent (if there are some elements in an array)

thoughts? (yes pure style but yet relying on implicit boolean coercion is error prone and not expressive I always find)

Reply:
I too prefer the more explicit approach, will change it back 👍